Behaviors/filewalker/FWAugmentation.js

```diff
@@ -0,0 +1,418 @@
+import { Behavior, BehaviorContext } from "@/Core/Behavior";
+import * as bt from "@/Core/BehaviorTree";
+import common from "@/Core/Common";
+import spell from "@/Core/Spell";
+import { me } from "@/Core/ObjectManager";
+import Settings from "@/Core/Settings";
+import EvokerCommon from "@/Behaviors/Nuclear/Retail/Evoker/EvokerCommon";
+import { defaultCombatTargeting as Combat, defaultCombatTargeting as combat } from "@/Targeting/CombatTargeting";
+import { PowerType } from "@/Enums/PowerType";
+import Specialization from "@/Enums/Specialization";
+
+// STATUS : TESTED, NOT OPTIMIZED
+
+const auras = {
+  prescience: 410089,
+  ebonMight: 395152,
+  ebonMightBuff: 395296,
+  breathOfEons: 403631,
+  blisteringScales: 360827, // Fixed typo in the name
+  tipTheScales: 370553,
+  chronomatic: 462497,
+  chronoflamesProc: 409560,
+  verdantEmbrace: 360995,
+  hover: 358267,
+  temporal: 424488, // Temporal Burst buff from Tip the Scales
+  ebonMightDamageBonus: 395296, // Individual buff received from Ebon Might
+  severing: 404756, // New Severing Slash
+  visionsOfObsidian: 408081, // Legendary talent
+  bountifulBloom: 370886 // May be useful for some builds
+};
+
+export class EvokerAugmentationBehavior extends Behavior {
+  name = "FW Augmentation Evoker";
+  context = BehaviorContext.Any;
+  specialization = Specialization.Evoker.Augmentation;
+  version = wow.GameVersion.Retail;
```

Comment:
version was removed in a recent commit and can be safely removed from all behaviors.